src/routes/admin/menu/+page.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+  import { categories, itemsByCategory } from "$stores/menuStore";
+  import type { Category, Item } from "$lib/types";
+</script>
+
+<div class="flex flex-col gap-8 overflow-y-auto">
+  <div class="flex flex-col gap-2">
+    <a href="/admin/menu/customization" class="btn w-full">Endre tilpasninger</a>
+    <a href="/admin/menu/item/new" class="btn w-full">Opprett et nytt produkt</a>
```

Comment:
```suggestion
    <a href="/admin/menu/customization" class="btn w-full">Rediger tilpasninger</a>
    <div class="divider"></div>
    <h1 class="mb-8 text-2xl">Produkter</h1>
    <a href="/admin/menu/item/new" class="btn w-full">Opprett et nytt produkt</a>
```

Prøve å skille endre tilpasninger knappen mer ut fra de to andre og gjøre den mer synlig.